apps/arena_load_test/lib/arena_load_test/socket_supervisor.ex

```diff
@@ -31,9 +32,17 @@ defmodule ArenaLoadTest.SocketSupervisor do
 
   # Creates `num_clients` clients to join a game
   def spawn_players(num_clients) do
-    for i <- 1..num_clients do
-      {:ok, _pid} = add_new_client(i)
+    case :ets.whereis(:clients) do
```

Comment:
this is the same as an if statement 
```
if :ets.whereis(:clients) == :undefinded do
 :ets.new(:clients, [:set, :named_table, :public])
end
```

Reply:
Yeah, I just wanted to make more explicit that when is not undefined is because there's no table called `:clients`